packages/elements/src/internals/machines/sign-up/machines/router.machine.ts

```diff
@@ -68,8 +68,8 @@ export const SignUpRouterMachine = setup({
     transfer: ({ context }) => context.router?.push(context.clerk.buildSignInUrl()),
   },
   guards: {
-    areFieldsMissing: ({ context }) => context.clerk.client.signUp.missingFields.length > 0,
-    areFieldsUnverified: ({ context }) => context.clerk.client.signUp.unverifiedFields.length > 0,
+    areFieldsMissing: ({ context }) => context.clerk?.client?.signUp?.missingFields?.length > 0,
```

Comment:
Might need to be a bit more thorough with applying optional chaining here. Will do another pass.